src/components/TokenIcon.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { render } from '@testing-library/react-native'
+import React from 'react'
+import { Provider } from 'react-redux'
+import TokenIcon from 'src/components/TokenIcon'
+import { createMockStore } from 'test/utils'
+import { mockCeloAddress, mockCusdAddress, mockTokenBalances } from 'test/values'
+
+// Setting up the mock token balances with expected additional values
+mockTokenBalances[mockCusdAddress].networkIconUrl =
+  'https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/CELO.png'
+
+const CELO_TOKEN = mockTokenBalances[mockCeloAddress]
+const CUSD_TOKEN = mockTokenBalances[mockCusdAddress]
```

Comment:
```suggestion
const CELO_TOKEN = mockTokenBalances[mockCeloAddress]
const CUSD_TOKEN = {
  ...mockTokenBalances[mockCusdAddress],
  networkIconUrl: 'https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/CELO.png'
```
so this doesn't change this globally.